tui/tui.go

```diff
@@ -77,6 +80,25 @@ type UI struct {
 	exec            func(argv0 string, argv []string, envv []string) error
 }
 
+func Execute(argv0 string, argv []string, envv []string) error {
+	// Windows does not support exec syscall.
+	if runtime.GOOS == "windows" {
+		cmd := exec.Command(argv0, argv...)
+		cmd.Stdout = os.Stdout
+		cmd.Stderr = os.Stderr
+		cmd.Stdin = os.Stdin
+		cmd.Env = envv
+		err := cmd.Run()
+		if err == nil {
+			os.Exit(0)
+		}
+		return err
+	}
```

Comment:
I prefer using [build constraints](https://golang.org/pkg/go/build/#hdr-Build_Constraints) for platform specific code. You do not ship unnecessary code then. Std lib does it the same way. Could you refactor it?

I would create files `tui/exec_windows.go` and `tui/exec_unix.go` and paste the implementations of `Execute` function there.